### PR TITLE
Add zh-TW translation

### PR DESCRIPTION
**Pull request by david082321**

### CODE REVIEW

**Comment by RodoMa92** [comment]
Can you please add your language in this [line](https://github.com/SteamDeckHomebrew/decky-loader/blob/main/frontend/i18next-parser.config.mjs#L39), so it will be automatically exported each time new strings are added?

File: backend/locales/zh-TW.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      "testing": "測試"
+    }
+  },
+  "Updater": {
+    "decky_updates": "Decky 更新",
+    "no_patch_notes_desc": "這個版本沒有更新日誌",
+    "patch_notes_desc": "更新日誌",
+    "updates": {
+      "check_button": "檢查更新",
+      "checking": "'正在檢查",
```
Comment (RodoMa92) [inline backend/locales/zh-TW.json:174]: The ' after the " was a typo in my english translation, after that is removed it looks good to merge to me.

**Review by RodoMa92 — CHANGES_REQUESTED** [review]
See my comment in the code below.

**Review by RodoMa92 — APPROVED** [review]
LGTM

**Comment by RodoMa92** [comment]
Merged it myself in the weblate instance that should almost ready to help translation, you can find it here if you need to fix my mistakes during copy and paste :P : https://weblate.werwolv.net/projects/decky/decky/

I think we can close this.

**Comment by david082321** [comment]
> Merged it myself in the weblate instance that should almost ready to help translation, you can find it here if you need to fix my mistakes during copy and paste :P : https://weblate.werwolv.net/projects/decky/decky/
> 
> I think we can close this.

I can't translate these two strings:
1. https://weblate.werwolv.net/translate/decky/decky/zh_Hant/?checksum=e89b0404c593bfe0
2. https://weblate.werwolv.net/translate/decky/decky/zh_Hant/?checksum=52c080da9da21b4d

**Pull request closed without merging**